### PR TITLE
add app vendor management

### PR DESCRIPTION
**Pull request by huojiao2006**

### CODE REVIEW

File: cmd/app-manager/main.go [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a Apache license
 // that can be found in the LICENSE file.
 
-// openpitrix app manager
+
```
Comment (zheng1) [inline cmd/app-manager/main.go]: this change is not necessary

File: pkg/service/appvendor/handler.go [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	vendor, err := GetVendorVerifyInfo(ctx, userID)
+
+	if vendor == nil && err != nil {
+		vendor = &models.VendorVerifyInfo{}
+		vendor.UserId = userID
+		vendor.Status = constants.StatusNew
+		vendor.StatusTime = time.Now()
+		userID, err = CreateVendorVerifyInfo(ctx, *vendor)
+		if err != nil {
+			logger.Error(ctx, "vendorVerifyInfo does not exit,create new vendorVerifyInfo failed [%s], %+v", userID, err)
```
Comment (zheng1) [inline pkg/service/appvendor/handler.go]: space after commas

File: pkg/service/appvendor/handler.go [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+			return nil, gerr.NewWithDetail(ctx, gerr.Internal, err, gerr.ErrorCreateResourceFailed)
+		}
+		logger.Debug(ctx, "vendorVerifyInfo does not exit,create new vendorVerifyInfo verify info, [%+v]", vendor)
```
Comment (zheng1) [inline pkg/service/appvendor/handler.go]: space after commas

File: pkg/service/appvendor/handler.go [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+		}
+	} else {
+		vendor := &models.VendorVerifyInfo{}
+		vendor = vendor.ParseReq2Vendor(req)
+		userID, err = CreateVendorVerifyInfo(ctx, *vendor)
+		if err != nil {
+			logger.Error(ctx, "Failed to submit vendorVerifyInfo [%+v], %+v", vendor, err)
+			return nil, gerr.NewWithDetail(ctx, gerr.Internal, err, gerr.ErrorCreateResourcesFailed)
+		}
+		logger.Debug(ctx, "vendorVerifyInfo does not exit,create new vendorVerifyInfo verify info, [%+v]", vendor)
```
Comment (zheng1) [inline pkg/service/appvendor/handler.go]: space after commas

File: cmd/app-manager/main.go (hunk 1 above)
Comment (huojiao2006) [reply cmd/app-manager/main.go]: done

File: pkg/service/appvendor/handler.go (hunk 2 above)
Comment (huojiao2006) [reply pkg/service/appvendor/handler.go]: done

File: pkg/service/appvendor/handler.go (hunk 3 above)
Comment (huojiao2006) [reply pkg/service/appvendor/handler.go]: done

File: pkg/service/appvendor/handler.go (hunk 4 above)
Comment (huojiao2006) [reply pkg/service/appvendor/handler.go]: done

**Pull request merged**